src/main/scala/nl/knaw/dans/easy/license/app/LicenseCreatorService.scala

```diff
@@ -114,24 +111,23 @@ class LicenseCreatorServlet extends ScalatraServlet with ApplicationSettings wit
         new InitialLdapContext(env, null)
       })
 
+    var success = false // TODO: get rid of var
```

Comment:
Your solution looked better, but did you test it? I couldn't get it to work, so for now, I've just rolled it back to ugly - but working - code.